tests/migration/Core/V6_7/Migration1697112043TemporaryPaymentAndShippingTechnicalNamesTest.php

```diff
@@ -0,0 +1,152 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Tests\Migration\Core\V6_7;
+
+use Doctrine\DBAL\ArrayParameterType;
+use Doctrine\DBAL\Connection;
+use PHPUnit\Framework\Attributes\After;
+use PHPUnit\Framework\Attributes\CoversClass;
+use PHPUnit\Framework\TestCase;
+use Shopware\Core\Defaults;
+use Shopware\Core\Framework\Log\Package;
+use Shopware\Core\Framework\Test\TestCaseBase\KernelTestBehaviour;
+use Shopware\Core\Framework\Uuid\Uuid;
+use Shopware\Core\Migration\V6_7\Migration1697112043TemporaryPaymentAndShippingTechnicalNames;
+use Shopware\Core\Test\Stub\Framework\IdsCollection;
+
+/**
+ * @internal
+ */
+#[Package('checkout')]
+#[CoversClass(Migration1697112043TemporaryPaymentAndShippingTechnicalNames::class)]
+class Migration1697112043TemporaryPaymentAndShippingTechnicalNamesTest extends TestCase
+{
+    use KernelTestBehaviour;
+
+    private Connection $connection;
+
+    private Migration1697112043TemporaryPaymentAndShippingTechnicalNames $migration;
+
+    private IdsCollection $ids;
+
+    protected function setUp(): void
+    {
+        $this->connection = static::getContainer()->get(Connection::class);
+        $this->migration = new Migration1697112043TemporaryPaymentAndShippingTechnicalNames();
+        $this->ids = new IdsCollection();
+    }
+
+    #[After]
+    public function cleanUp(): void
+    {
+        $byteList = Uuid::fromHexToBytesList(\array_values($this->ids->all()));
+
+        foreach (['payment_method', 'shipping_method', 'app_shipping_method', 'app_payment_method'] as $table) {
+            $this->connection->executeStatement(
+                'DELETE FROM `' . $table . '` WHERE `id` in (:ids)',
+                ['ids' => $byteList],
+                ['ids' => ArrayParameterType::BINARY],
+            );
+        }
+    }
+
+    public function testMigrate(): void
+    {
+        $this->rollback();
+        $this->prepare();
```

Comment:
please use database transaction instead. We're not altering tables here.

Reply:
It would be lovely it that's true, but I need to roll back to nullable `technical_name` fields